ng_will_paginate.gemspec

```diff
@@ -24,4 +24,6 @@ Gem::Specification.new do |spec|
 
   spec.add_development_dependency "bundler", "~> 1.8"
   spec.add_development_dependency "rake", "~> 10.0"
+  spec.add_runtime_dependency "will_paginate"
+  
```

Comment:
Note to future selves: This doesn't seem to be working. Right now, client app needs to add 'will_paginate' to their Gemfile. We'll figure it out.